docs/contents/services.md

```diff
@@ -1,26 +1,149 @@
-|Amazon [API Gateway V2](https://aws.amazon.com/api-gateway/)|`DEVELOPER PREVIEW`|[`apigatewayv2`](https://github.com/aws/aws-controllers-k8s/tree/main/services/apigatewayv2)|
-|Amazon [DynamoDB](https://aws.amazon.com/dynamodb/)|`DEVELOPER PREVIEW`|[`dynamodb`](https://github.com/aws/aws-controllers-k8s/tree/main/services/dynamodb)|
-|Amazon [ECR](https://aws.amazon.com/ecr/)|`DEVELOPER PREVIEW`|[`ecr`](https://github.com/aws/aws-controllers-k8s/tree/main/services/ecr)|
-|Amazon [S3](https://aws.amazon.com/s3/)|`DEVELOPER PREVIEW`|[`s3`](https://github.com/aws/aws-controllers-k8s/tree/main/services/s3)|
-|Amazon [SQS](https://aws.amazon.com/sqs/)|`BUILD`|`sqs`|
-|Amazon [SNS](https://aws.amazon.com/sns/)|`DEVELOPER PREVIEW`|[`sns`](https://github.com/aws/aws-controllers-k8s/tree/main/services/sns)|
-|Amazon [ElastiCache](https://aws.amazon.com/elasticache/)|`DEVELOPER PREVIEW`|[`elasticache`](https://github.com/aws/aws-controllers-k8s/tree/main/services/elasticache)|
-
 !!! note "IMPORTANT"
     There is no single release of the ACK project. The ACK project contains a
     series of service controllers, one for each AWS service API. Each
     individual ACK service controller is released separately. Please see the
     documentation on [release criteria](releases.md) for information on how we
     release ACK service controllers.
 
-![ACK release criteria](images/release-criteria.png)
+| AWS Service | Current Status | Next Milestone
+| ----------- | -------------- | --------------
+| Amazon [API Gateway V2][apigwv2] | `DEVELOPER PREVIEW` | [`BETA`](https://github.com/aws/aws-controllers-k8s/milestone/15)
+| Amazon [CloudFront Distribution][cfd] | `PLANNED` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/14)
+| Amazon [DynamoDB][dynamodb] | `DEVELOPER PREVIEW` |
+| Amazon [ECR][ecr] | `DEVELOPER PREVIEW` |
+| Amazon [EFS][efs] | `PROPOSED` |
+| Amazon [EKS][eks] | `PLANNED` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/7)
+| Amazon [ElastiCache][elasticache] | `DEVELOPER PREVIEW` | [`BETA`](https://github.com/aws/aws-controllers-k8s/milestone/9)
+| Amazon [Lambda][lambda] | `PLANNED` |
+| Amazon [MQ][mq] | `PLANNED` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/12)
+| Amazon [MSK][msk] | `PLANNED` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/13)
+| Amazon [RDS][rds] | `PLANNED` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/8)
+| Amazon [Sagemaker][sagemaker] | `BUILD` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/11)
+| Amazon [SNS][sns] | `DEVELOPER PREVIEW` |
+| Amazon [SQS][sqs] | `BUILD` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/6)
+| Amazon [StepFunctions][sfn] | `BUILD` | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/10)
+| Amazon [S3][s3] | `DEVELOPER PREVIEW` |
+
```

Comment:
I'm curious about plans for IAM.  There's https://github.com/aws/aws-controllers-k8s/issues/222, but does it make sense to add something here too?

Reply:
@dbyron0 yes :) As soon as this PR is merged, feel like pushing a PR that adds that?

Reply:
Sure, if `PLANNED` is the right value for the `Current Status` column, with `Next Milestone` blank.

Reply:
It would be PROPOSED for IAM, since we don't yet have set plans for it :)

Reply:
Like [this](https://github.com/aws/aws-controllers-k8s/pull/495)?  I also saw https://github.com/aws/aws-controllers-k8s/issues/484 fly by.